src/Strategy/JsonStrategy.php

```diff
@@ -70,7 +70,7 @@ public function getExceptionDecorator(Exception $exception)
             ]));
 
             $response = $response->withAddedHeader('content-type', 'application/json');
-            return $response->withStatus(500, $exception->getMessage());
+            return $response->withStatus(500, strtok($exception->getMessage(), "\n"));
```

Comment:
I never heard about this method `strtok`. So I learned something new today. Thank you for that.